### DISCUSSION TITLE
Doris on Elasticsearch 的对象数据类型怎么匹配？

### DISCUSSION THREAD

**Discussion by kirning** [apache/doris#13632] [A - General / Q&A]
我看了一下官方的列表

Doris\ES | byte | short | integer | long | float | double | keyword | text | date
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
tinyint | √ |   |   |   |   |   |   |   |  
smallint | √ | √ |   |   |   |   |   |   |  
int | √ | √ | √ |   |   |   |   |   |  
bigint | √ | √ | √ | √ |   |   |   |   |  
float |   |   |   |   | √ |   |   |   |  
double |   |   |   |   |   | √ |   |   |  
char |   |   |   |   |   |   | √ | √ |  
varchar |   |   |   |   |   |   | √ | √ |  
date |   |   |   |   |   |   |   |   | √
datetime |   |   |   |   |   |   |   |   | √

并没有提到对象类型的数据应该如何对应，我应该怎么处理？

**Comment by stalary** [comment]
https://github.com/apache/doris/blob/master/docs/zh-CN/docs/ecosystem/external-table/multi-catalog.md#elasticsearch
